### PR TITLE
feat: Localizations for toasts caused by duplicate mod folders

### PR DESCRIPTION
**Pull request by 0neGal**
This PR is for merging localization of the new duplicate mod toasts.

#### New localization keys

Note that the second one key has the mod name prefixed.

```json
"gui.toast.title.duped": "Duplicate folder names!",
"gui.toast.desc.duped": "has multiple mod folders in it, with the same name, causing duplicate folders, if you're the developer, you should fix this.",
```

#### Maintainers

 - [x] German - @DxsSucuk
 - [x] French - @Alystrasz
 - [x] Spanish - @AA-Delta

### CODE REVIEW

**Comment by DxsSucuk** [comment]
Done check https://github.com/0neGal/viper/pull/156

File: src/lang/de.json [hunk 1]
```diff
@@ -134,9 +134,11 @@
 	"gui.toast.title.installed": "Mod installiert!",
 	"gui.toast.title.failed": "Fehler beim Installieren des Mods!",
 	"gui.toast.title.malformed": "Fehlerhafte Ordnerstruktur!",
+	"gui.toast.title.duped": "Duplizierter Ordner name!",
 	"gui.toast.desc.installed": "wurde installiert!",
 	"gui.toast.desc.malformed": "hat eine fehlerhafte Ordnerstruktur, falls du der Entwickler bist, solltest du dies beheben.",
 	"gui.toast.desc.failed": "Ein unbekannter Fehler ist aufgetaucht beim Installieren, die Schuld kann beim Autor liegen oder bei Viper selbst!",
+	"gui.toast.desc.duped": "hat multiple Ordner in sich mit dem selben Namen, wodurch ein duplizierter Ordner ensteht! Falls du der Entwickler bist solltest du dies beheben!",
```
Comment (GeckoEidechse) [inline src/lang/de.json:141]:
```suggestion
	"gui.toast.desc.duped": "hat mehrere Ordner in sich mit dem selben Namen, wodurch ein duplizierter Ordner ensteht! Falls du der Entwickler bist solltest du dies beheben!",
```

"mehrere" sounds better than "multiple", the latter isn't even a proper German word ^^

Source: I speak German :P

Comment (DxsSucuk) [reply src/lang/de.json:141]: "multiple" Indeed isn't german but it is still being used in the german language.
We often refer to it as "Eingedeutschtes Wort", which basically just means this Word is being used in german sentences even thou it is not german. (Or we change up the word from another language and add it to german)

Comment (0neGal) [reply src/lang/de.json:141]: I don't speak a word of German, so I don't know which German to trust here...

Comment (DxsSucuk) [reply src/lang/de.json:141]: Both are okay but I have to say that @GeckoEidechse solution sounds better and would be easier to understand for users.

**Pull request merged**